docs/MAZERUNNER.md

```diff
@@ -0,0 +1,48 @@
+# Mazerunner tests
+
+E2E tests are implemented with our notifier testing tool [Maze runner](https://github.com/bugsnag/maze-runner), which is a black-box test framework written in Ruby.
+
+## Running remote end-to-end tests
+
+End to end tests are written in cucumber-style `.feature` files, and need Ruby-backed "steps" in order to know what to run. The tests are located in the top level [`features`](/features/) directory.
+
+Maze Runner's CLI and the test fixtures are containerized so you'll need Docker (and Docker Compose) to run them.
```

Comment:
A docker image is provided for Maze Runner, but you don't have to use it and have docker/compose installed.  You can quite happily run it as a Gem (and indeed that is what the later instruction here do).  So I think you can simply remove this line, or just note that Ruby and Bundler are required.

Reply:
Removed this line.